omod-1.8/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs1_8/ModuleActionResource1_8.java

```diff
@@ -228,7 +228,7 @@ private void startModules(Collection<Module> modules, ServletContext servletCont
 		}
 		
 		for (Module module : modules) {
-			if (moduleFactoryWrapper.isModuleStopped(module)) {
+			if ((moduleFactoryWrapper.isModuleStopped(module)&& module.getModuleId()).equal( existingModule.getModuleId())) {
```

Comment:
aren't we changing the meaning of the original line by adding this change?

Reply:
@HerbertYiga  i have made changes  kindly check in commit  thank you